docs/spec_draft.md

```diff
@@ -1420,6 +1421,74 @@ Produces an `output` tensor from a constant `value`.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.convert
+
+### Semantics
+
+Performs an element-wise conversion of values similar to an element-wise
+`static_cast` in C++ from one element type to another on `operand` tensor and
+produces a `result` tensor.
+
+For conversions involving integer-to-integer, if there is an unsigned/signed
+overflow, the result is implementation-defined and one of the following:
+  * mathematical result modulo $2^n$, where n is the bit width of the result,
+    for unsigned overflow. For signed integer overflow, wraps the result around
+    the representable range $[-2^{n-1},\ 2^{n-1} - 1]$.
+  * saturation to $2^{n-1} - 1$ (or $-2^{n-1}$) for signed overflow and
+    saturation to $2^n - 1$ (or $0$) for unsigned overflow.
+
+For conversions involving floating-point-to-floating-point or
+complex-to-complex, if the source value can be exactly represented in the
+destination type, the result value is that exact representation. If the source
+value is between two adjacent destination values, the result is
+implementation-defined. Otherwise, the behavior is undefined.
+
+For conversions involving floating-point-to-complex or
+complex-to-floating-point, the imaginary value is zero or truncated,
+respectively. The conversion of inner representation of complex element type
+follows the floating-point-to-floating-point conversion.
+
+For conversions involving floating-point-to-integer or complex-to-integer,
```

Comment:
Reword:
For conversions involving **floating-point-to-integer**,
the fractional part is truncated. The behavior is undefined if the truncated
value cannot be represented in the destination type. Conversions involving **complex-to-integer** follows the same behavior while converting the source real part to destination integer. Additionally, the source imaginary part is ignored.